.gitignore

```diff
@@ -56,3 +56,6 @@ terraform.tfstate
 # ansible stuff
 ssh.config
 *.retry
+
+# vim-specific files
+*.swp
```

Comment:
I would move this up to the "editors" section above where `.vscode` is already added to keep everything together.